erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -661,7 +665,7 @@ def validate_proj_cust(self):
 				throw(_("Customer {0} does not belong to project {1}").format(self.customer,self.project))
 
 	def validate_pos(self):
-		if self.is_return:
+		if self.is_return and not self.is_pos:
```

Comment:
This validation was specifically for `is_pos` invoices. This shouldn't be bypassed.